src/python/pants/option/global_options.py

```diff
@@ -693,6 +693,14 @@ def register_bootstrap_options(cls, register):
             ),
             default=os.path.join(get_pants_cachedir(), "named_caches"),
         )
+        register(
+            "--ca-certs-path",
+            advanced=True,
+            type=str,
+            default=None,
+            help="Path to a file containing PEM-format CA certificates used for verifying secure "
+            "connections when downloading files required by a build.",
```

Comment:
Does this path have to be in the build root?

Reply:
No.